src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -146,4 +175,52 @@ export class DotEditLayoutGridComponent implements OnInit {
         }
         return rowArray;
     }
+
+    private transformDataToDisplayOnGrid(resp): any {
```

Comment:
```resp``` is a ```pageView``` right?
doesn't this return something typed?

Reply:
this shpuld return a LayoutGridBox[] (I think)